src/main/java/core/levels/LevelLoader.java

```diff
@@ -2,11 +2,26 @@
 
 import com.badlogic.gdx.math.Vector2;
 import com.badlogic.gdx.physics.box2d.World;
+import core.world.EntitySpawner;
+import static core.world.DemoSpawners.*;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class LevelLoader {
 
+    /**
+     * Load level with spawners and map
+     *
+     */
     public static LevelState getLevel1() {
-        LevelState lvl = new LevelState("maps/demo.tmx");
+        // Create enemy spawner list
+        List<EntitySpawner> enemies = new ArrayList<EntitySpawner>();
```

Comment:
I think this should be a private/protected attribute outside of this constructor for extendibility purposes